### DISCUSSION TITLE
arg can not show select list in terminal

### DISCUSSION THREAD

**Discussion by xxleyi** [johnlindquist/kit#779] [Errors] [answered]
![image](https://user-images.githubusercontent.com/12626454/172389551-caf29eef-4653-4d62-91f1-6dc249a3a21c.png)

work in UI mode

version: 1.17.20

**Comment by johnlindquist** [comment]
Thanks for reporting, I'll get this sorted in the next release

**Comment by johnlindquist** [comment, marked as answer]
Also, if you'd like it to work in this release, please use the name/value version of choices:


[Open testing-arg-in-terminal in Script Kit](https://scriptkit.com/api/new?name=testing-arg-in-terminal&url=https://gist.githubusercontent.com/johnlindquist/381624575c109da92cf8f284e1da89f4/raw/51629a42b24a49e4065a34cfab2a0d3e34d60e55/testing-arg-in-terminal.ts")

```js
// Name: Testing Arg in Terminal

import "@johnlindquist/kit"

await arg("Testing", [
    {name: "one", value: "one"},
    {name: "two", value: "two"}
])


```

**Reply by xxleyi** [reply]
Thanks your quick reply, it works :-)
